source/_integrations/enphase_envoy.markdown

```diff
@@ -99,3 +99,76 @@ This integration provides several values suitable for the energy dashboard:
 [^3]: Only applies when using  Envoy S Metered / IQ Gateway Metered with installed and configured current transformers (<abbr title="current transformers">CT</abbr>).
 
 There are no readily available battery energy sensors for use with the `Home Battery storage`. You can consider using the Encharge  `real_power_mw` entity as an input to Riemann integrators for charge (negative) or discharge (positive) values. As the [polling interval](#polling-interval) is 1 minute, these may be off though.
+
+## Debug logs and diagnostics
+
+This integration provides debug log and diagnostics report as described in the [Home Assistant troubleshooting pages](/docs/configuration/troubleshooting/#debug-logs-and-diagnostics).
+
+### Debug log
+
+When experiencing issues during use of the integration, enable the debug log for the Envoy / IQ Gateway. This will add details on the data collections to the Home Assistant log file. Leave the debug log enabled long enough to capture occurrence of the issue. If the issue is intermittent this may take a while and it may grow the log file quite a bit.
```

Comment:
```suggestion
When experiencing issues during the use of the integration, enable the debug log for the Envoy / IQ Gateway. This will add details on the data collection to the Home Assistant log file. Leave the debug log enabled long enough to capture the occurrence of the issue. If the issue is intermittent, this may take a while and it may grow the log file quite a bit.
```